sdk/datafactory/arm-datafactory/CHANGELOG.md

```diff
@@ -1,15 +1,225 @@
 # Release History
+    
+## 12.0.0 (2023-11-10)
+    
+**Features**
 
-## 11.1.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Added operation group ChangeDataCapture
```

Comment:
How as this changelog generated? In general, avoid these types of changelogs because they are not very useful.